lib/bucket_files.js

```diff
@@ -30,15 +30,32 @@ module.exports = (client, bucket, logger) => {
 
   const current = async () => {
     try {
-      logger.info(`executing listObjects for bucket: ${bucket}`)
-      const results = await client.listObjects({ Bucket: bucket }).promise()
-      logger.debug(`listObjects response for bucket: ${bucket}`, results)
-      logger.info(`listObjects returned ${results.Contents.length} for bucket: ${bucket}`)
-      return results.Contents
+        var params = { Bucket: bucket };
```

Comment:
Can you replace all the `var` references with `const` (or `let`) in the following code? Helps us to catch errors.